scripts/build.js

```diff
@@ -30,6 +31,14 @@ const runWebpack = config => {
   });
 };
 
+const cleanDistFolders = async () => {
+  await Promise.all(builds.map(({ paths }) => rimraf(paths.dist)));
+};
+
+const compileAll = async () => {
```

Comment:
We can just return `Promise.all` here; this doesn't need to be an `async` function.